src/components/Content.tsx

```diff
@@ -7,13 +7,16 @@ import { format } from 'date-fns';
 const styles = {
   container: {
     padding: 60,
+    minWidth: 500,
+    boxShadow: '-2px 0 12px 0 rgba(0,0,0,0.1)',
+    position: 'relative',
   },
 };
 
 const DATE_KEYS = ['createdAt', 'lastLogin'];
-export default ({ data, keys }) => {
+export default ({ data, keys, styles: stylesProp = {} }) => {
```

Comment:
could this just be `style` instead of styles and renaming and having a default?

Reply:
want passing down of styles to be consistent, always `styles`

Reply:
hmm, this seems inconsistent :D It doesn't match react `style` and in the code so far `styles` is an object with style objects in it, rather than a style object itself.

Reply:
I meant consistent between prop names, I believe I was using `styles` as the prop for other components too?
Wanted to intentionally differentiate from react's style prop, so that you know you can pass nested css rules, e.g.

```
styles={{
  '&:hover': { color: 'red' },
  '& .sub-component': { fontSize: 22 }
}}
```

Although maybe the distinction isn't important